irispy/new_sji.py

```diff
@@ -286,6 +286,42 @@ def dimensions(self):
     def world_axis_physical_types(self):
         return self.cube_like_world_axis_physical_types
 
+    def apply_exposure_time_correction(self, undo=False, force=False):
```

Comment:
The API should mimic that of ```IRISSpectrogramCubeSequence.apply_exposure_time_correction```.  So there should also be a ```copy=False``` kwarg.  Then in the method, the ```self.data``` should be overwritten is ```copy=False``` and a new object returned if ```copy=True```.  See https://github.com/sunpy/irispy/blob/master/irispy/spectrograph.py#L174